source/includes/run-command.kt

```diff
@@ -0,0 +1,23 @@
+import com.mongodb.kotlin.client.MongoClient
+import org.bson.Document
+
+fun main() {
+    //start-execute
+    val explanation = database.runCommand({
+            explain: {
+                find: 'restaurants'
+            }
+        })
+    //end-execute
+
+    //start-read-preference
+    val command = { "hello": 1 }
```

Comment:
Similar callout as above

```suggestion
    val command =  Document("hello", 1)

```